docs/standard/security/vulnerabilities-cbc-mode.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Timing vulnerabilities with CBC-mode symmetric decryption using padding"
+title: Detect timing vulnerabilities in CBC-mode symmetric decryption
 description: "Learn how to detect and mitigate timing vulnerabilities with Cipher-Block-Chaining (CBC) mode symmetric decryption using padding."
-ms.date: "06/12/2018"
-author: "blowdart"
-ms.author: "mairaw"
+ms.date: 06/12/2018
+author: blowdart
+ms.author: mairaw
```

Comment:
Double quotes can be removed from the description too. (Unless there is a known style that recommends adding it in the description.)

Reply:
That works here. Thanks!